satpy/etc/readers/nwcsaf-geo.yaml

```diff
@@ -370,7 +380,7 @@ datasets:
     file_type: nc_nwcsaf_crr-ph
 
   crrph_pal:
-    name: crrph_intensity_pal
+    name: crrph_pal
```

Comment:
Is this backwards compatible with v2018?

Reply:
This is a fix also for v2018, there's no `crrph_intensity_pal` in v2018 files.